### ISSUE TITLE
Add option to ignore accounts that have manually been unmuted/unblocked previously

### ISSUE THREAD

**Issue by cooljeanius** [kheina-com/Blue-Blocker#55]
Sometimes this extension makes a mistake and blocks (or in my case, mutes) accounts that I'd already unblocked/unmuted previously. I would like for this extension to remember unblocks/unmutes and then remember to not block/mute them again if it encounters them again (or, if this is already supposed to be how the extension behaves, it should be documented more clearly)

**Comment by DanielleMiu** [comment]
so, I hadn't even thought of doing this tbh. this was the general idea around adding the following bypass though. without sounding too rude, is there ever a situation where you want to manually unblock someone but not follow them? (or they aren't following you, since that's an option as well)

honestly, I just didn't consider this situation a possibility so please let me know!

**Comment by cooljeanius** [comment]
> is there ever a situation where you want to manually unblock someone but not follow them? 

Well, I'm stuck at the Follow Limit (of 5,000), so often times I *can't* follow people even if I want to...

**Comment by DanielleMiu** [comment]
wait wtf, there's a follow limit now???

**Comment by cooljeanius** [comment]
> wait wtf, there's a follow limit now???

There's been one for awhile; I hit it back in 2018 myself...

**Comment by DanielleMiu** [comment]
I'll be looking into this.

**Comment by DanielleMiu** [comment]
closing in favor of https://github.com/kheina-com/Blue-Blocker/issues/59 (sorry, I'm adding more detail to it)
